android/src/app/organicmaps/bookmarks/BookmarkCategoriesAdapter.java

```diff
@@ -154,8 +154,23 @@ public void onBindViewHolder(final RecyclerView.ViewHolder holder, final int pos
   private void bindSize(@NonNull CategoryViewHolder categoryHolder,
                         @NonNull BookmarkCategory category)
   {
-    BookmarkCategory.CountAndPlurals template = category.getPluralsCountTemplate();
```

Comment:
Is CountAndPlurals used somewhere else? Can it be removed?

Reply:
It also is used in collections, which I can't find while using the app. Started here, but not sure if it is in use now:

https://github.com/organicmaps/organicmaps/commit/0008eb11

Reply:
@rtsisyk As it is used also in collections, it's better to leave it as is to avoid code duplication. It can be refactored in the future separately, if necessary.

Btw, the code here is more verbose than the CountAndPlurals class.

Reply:
I assume that CountAndPlurals should also be updated for collections to handle the same situation as handled here, right? E.g. the setDetailedSize method should be added there, and reused, right?

Reply:
@biodranik, @rtsisyk, could you please suggest, where could be collections found on UI?

Reply:
They are turned off temporarily. We likely enable them back later.